spec/dummy/app/models/invoice.rb

```diff
@@ -92,4 +92,18 @@ def self.colors
   def set_default_state
     self.state ||= 'pending'
   end
+
+  def self.ransackable_attributes(auth_object = nil)
+    [
+      "aasm_state", "active", "amount", "attachment_content_type", "attachment_file_name",
+      "attachment_file_size", "attachment_updated_at", "category_id", "city_id", "client_id",
+      "color", "created_at", "description", "id", "legal_date", "number", "paid",
+      "photo_content_type", "photo_file_name", "photo_file_size", "photo_updated_at",
+      "picture_data", "position", "shipping_status", "state", "status", "updated_at",
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Style/TrailingCommaInArrayLiteral: Avoid comma after the last item of an array.